### PR TITLE
Fix python wrappers for BetweenFactorVector and PreintegratedCombined…

### PR DESCRIPTION
**Pull request by tardifjp**
…Measurements and

added Vector type to the BetweenFactor generation
Created PreintegrationCombinedParams to add constructor for PreintegratedCombinedMeasurements

---
This change is [Reviewable](https://reviewable.io/reviews/borglab/gtsam/153)

### CODE REVIEW

File: gtsam/navigation/CombinedImuFactor.h [hunk 1]
```diff
@@ -54,6 +48,61 @@ typedef ManifoldPreintegration PreintegrationType;
...
+    ar& BOOST_SERIALIZATION_NVP(biasOmegaCovariance);
+    ar& BOOST_SERIALIZATION_NVP(biasAccOmegaInt);
+  }
+
+public:
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+};
+
+  
+#ifdef GTSAM_TANGENT_PREINTEGRATION
```
Comment (dellaert) [inline gtsam/navigation/CombinedImuFactor.h]: Is there a reason this moved? Maybe move it back as it seems orthogonal

File: gtsam.h [hunk 2]
```diff
@@ -2812,7 +2812,34 @@ virtual class ImuFactor: gtsam::NonlinearFactor {
 };
 
 #include <gtsam/navigation/CombinedImuFactor.h>
+//typedef PreintegratedCombinedMeasurements::Params PreintegrationCombinedParams;
```
Comment (dellaert) [inline gtsam.h]: kill :-)

File: gtsam/navigation/CombinedImuFactor.h [hunk 3]
```diff
@@ -54,6 +48,61 @@ typedef ManifoldPreintegration PreintegrationType;
  *     Robotics: Science and Systems (RSS), 2015.
  */
 
+struct GTSAM_EXPORT PreintegrationCombinedParams : PreintegrationParams {
```
Comment (dellaert) [inline gtsam/navigation/CombinedImuFactor.h]: Add doxygen /** */ block above?

File: gtsam.h (hunk 2 above)
Comment (tardifjp) [reply gtsam.h]: Yes, that's from when I was trying to leave CombinedImuFactor alone!

File: gtsam/navigation/CombinedImuFactor.h (hunk 1 above)
Comment (tardifjp) [reply gtsam/navigation/CombinedImuFactor.h]: Ok, I moved it back to the top

File: gtsam/navigation/CombinedImuFactor.h (hunk 3 above)
Comment (tardifjp) [reply gtsam/navigation/CombinedImuFactor.h]: ok, added a quick description of the new struct

**Comment by dellaert** [comment]
Awesome. Will merge when check passes.
Thanks again !!!!

**Comment by tardifjp** [comment]
One last thing I haven't pushed. In CombinedImuFactor.h, I tried changing:
    ar& BOOST_SERIALIZATION_BASE_OBJECT_NVP(PreintegratedRotation::Params);
for
    ar& BOOST_SERIALIZATION_BASE_OBJECT_NVP(PreintegrationParams);
in the serialize function.

It would seem logical since that's the actual superclass. It compiles just fine. However, I don't have a way to validate that change.

**Comment by dellaert** [comment]
> One last thing I haven't pushed. In CombinedImuFactor.h, I tried changing:
> ar& BOOST_SERIALIZATION_BASE_OBJECT_NVP(PreintegratedRotation::Params);
> for
> ar& BOOST_SERIALIZATION_BASE_OBJECT_NVP(PreintegrationParams);
> in the serialize function.
> 
> It would seem logical since that's the actual superclass. It compiles just fine. However, I don't have a way to validate that change.

@tardifjp I will merge, but could you do one last thing for us and open up an issue for the serialization bug. It is a bug, and you found it. 

To verify it you would use a serialization test to and from a string. So, adding a unit test...

**Comment by tardifjp** [comment]
Sure, I'll create the issue.
